stingray/powerspectrum.py

```diff
@@ -205,16 +213,18 @@ def compute_rms(self, min_freq, max_freq):
         minind = self.freq.searchsorted(min_freq)
         maxind = self.freq.searchsorted(max_freq)
         powers = self.power[minind:maxind]
+        nphots = self.nphots
 
         if self.norm.lower() == 'leahy':
-            rms = np.sqrt(np.sum(powers) / self.nphots)
-
+            powers_leahy = powers.copy()
         elif self.norm.lower() == "frac":
-            rms = np.sqrt(np.sum(powers * self.df))
+            powers_leahy = \
+                self.unnorm_power[minind:maxind].real * 2 / nphots
         else:
             raise TypeError("Normalization not recognized!")
```

Comment:
Better to include the value of self.norm in the error message of the exception/error.